test/iht/controllers/registration/RegistrationSummaryControllerTest.scala

```diff
@@ -100,11 +100,11 @@ class RegistrationSummaryControllerTest extends RegistrationControllerTest{
 
       val result = controller.onPageLoad()(createFakeRequest())
       status(result) should be(OK)
-      val content = contentAsString(result)
+      val content = contentAsString(result).replace("\n","")
```

Comment:
Is this replace necessary? If so, it might be better to do the replace in a method rather than duplicating this code.